OpenSource-Strategien-Uebersicht.md

```diff
@@ -29,35 +29,35 @@ Im Sinne einer nüchternen Betrachtung verlasse ich jetzt das Bierzelt und komme
+Freie Software, oder Open Source Software, wie wir sie technisch heute nennen, ist aber kein Einzelfall. Sie ist allgegenwärtig und in einem umfassenden Leistungsspektrum mit hohem Qualitätsstandard verfügbar. Freie Software hat sich über die vergangenen Jahrzehnte kontinuierlich zu einer Digitalen Allmende entwickelt. Freie Software öffnet als Gemeingut einen kreativen und produktiven Raum neben den klassischen Märkten für Lizenzen und proprietären Produkte.
 
-Anders als die mittelalterliche Gemeinweide ist die Digitale Allmende resistent gegen Übernutzung. Die Open Source Lizenzen sichern den Fortbestand und das Wachstum selbst wenn ein Großteil der Nutzer als Trittbrettfahrer unterwegs sind.
-Damit hat Freie Software nach und nach die Struktur des Software Geschäfts insgesamt verändert. Das Server Geschäft wird von Linux dominiert, die Smartphones von Android, Webserver sind Freie Software, Programmiersprachen und Entwicklungsumgebungen, ein Großteil der Datenbanken, Dokument Managementsysteme, AI und ML Frameworks alles ist vielfältig und in hoher Qualität als Freie Software erhältlich.
+Anders als die mittelalterliche Gemeinweide ist die Digitale Allmende resistent gegen Übernutzung. Die Open Source Lizenzen sichern den Fortbestand und das Wachstum, selbst wenn ein Großteil der Nutzer als Trittbrettfahrer unterwegs ist.
+Damit hat Freie Software nach und nach die Struktur des Software-Geschäfts insgesamt verändert. Das Server-Geschäft wird von Linux dominiert, die Smartphones von Android, Webserver sind Freie Software, Programmiersprachen und Entwicklungsumgebungen, ein Großteil der Datenbanken, Dokumenten-Management-Systeme, AI und ML Frameworks, alles ist vielfältig und in hoher Qualität als Freie Software erhältlich.
 
 Natürlich gibt es weiterhin viele kommerzielle Software-Angebote. Die positionieren sich aber immer in einem Umfeld von Freier Software. Rein proprietäre Angebote werden verdrängt.
 
 ### Exkurs: Commons
 Das Wort Allmende
-bezeichnet ursprünglich das gemeinschaftlich genutzte Land einer Dorfgemeinschaft, das Gemeingut. Im englischen ist dafür der Begriff [Commons](https://en.wikipedia.org/wiki/Elinor_Ostrom) gebräuchlich und zum Beispiel als Creative Commons auch im Kontext von Freier Software etabliert.
+bezeichnet ursprünglich das gemeinschaftlich genutzte Land einer Dorfgemeinschaft, das Gemeingut. Im Englischen ist dafür der Begriff [Commons](https://en.wikipedia.org/wiki/Commons) gebräuchlich und zum Beispiel als Creative Commons auch im Kontext von Freier Software etabliert.
```

Comment:
Du hast natürlich Recht das Elinor Ostrom nicht einfach nur Commons repräsentiert. Sie hat aber den Nobelpreis in Ökonomie für ihre Arbeit über die Bewirtschaftung von Commons erhalten. Das ist mir hier als Link wichtig, auch oder grade weil ich es hier in dem Text nicht zusätzlich einführen will.
```suggestion
bezeichnet ursprünglich das gemeinschaftlich genutzte Land einer Dorfgemeinschaft, das Gemeingut. Im Englischen ist dafür der Begriff [Commons](https://en.wikipedia.org/wiki/Elinor_Ostrom) gebräuchlich und zum Beispiel als Creative Commons auch im Kontext von Freier Software etabliert.
```